### PR TITLE
feat: mark as sold

### PR DESCRIPTION
**Pull request by Dimentium**
+ Added the ability to mark a published Ads as obsolette

### CODE REVIEW

File: send.go [hunk 1]
```diff
@@ -80,9 +87,27 @@ func (vbbot *VBBot) sendPlainMessage(update tgbotapi.Update) {
...
+	sentMsg, err := vbbot.tgbot.Send(msg)
 	if err != nil {
 		vbbot.handleError(err, update.Message.Chat.ID)
+		return
+	} else {
+		vbbot.confirmNewInzerat(update.Message.From.ID, update.Message.MessageID, sentMsg)
+	}
+}
+
+func (vbbot *VBBot) confirmNewInzerat(chatId int64, replyToId int, sentMsg tgbotapi.Message) {
```
Comment (timonomsk) [inline send.go]: Could we change name of this function to keep names in English language?

**Comment by golman** [comment]
the idea is great, but the implementation should be improved, I would like it not to replace the whole text of the ad, but to add that the item is sold, the text itself can be crossed out, for example.

File: telegram-bot.go [hunk 2]
```diff
@@ -78,8 +80,60 @@ func (vbbot *VBBot) Start(ch tgbotapi.UpdatesChannel) {
...
+	}
+
+	// Optionally, confirm the successful edit to the user
+	callback := tgbotapi.NewCallback(query.ID, "Всё!")
+	vbbot.tgbot.Request(callback)
+
+	editedMsg = tgbotapi.NewEditMessageText(query.Message.Chat.ID, query.Message.MessageID, "Продано!")
+	_, err = vbbot.tgbot.Send(editedMsg)
+
+	// deletedMsg := tgbotapi.NewDeleteMessage(query.Message.Chat.ID, query.Message.MessageID)
```
Comment (timonomsk) [inline telegram-bot.go]: It would be great to remove leftovers

File: send.go (hunk 1 above)
Comment (Dimentium) [reply send.go]: fixed

File: telegram-bot.go (hunk 2 above)
Comment (Dimentium) [reply telegram-bot.go]: fixed

**Comment by Dimentium** [comment]
> the idea is great, but the implementation should be improved, I would like it not to replace the whole text of the ad, but to add that the item is sold, the text itself can be crossed out, for example.

reworked

**Pull request closed without merging**